modules/ossm-rn-new-features.adoc

```diff
@@ -17,13 +17,175 @@ This release adds improvements related to the following components and concepts.
 
 == New features {SMProductName} version {SMProductVersion}
 
-This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), bug fixes, and is supported on OpenShift Container Platform 4.9 or later.
+This release of {SMProductName} introduces new features, addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on OpenShift Container Platform 4.9, 4.10, and 4.11.
 
 === Component versions included in {SMProductName} version {SMProductVersion}
 
 |===
 |Component |Version
 
+|Istio
+|1.14
+
+|Envoy Proxy
+|1.22.4
+
+|Jaeger
+|1.38
+
+|Kiali
+|1.57.3
+|===
+
+=== New Container Network Interface (CNI) DaemonSet container and configuration map
+
+The `openshift-operators` namespace includes a new istio CNI DaemonSet `istio-cni-node-v2-3` and a new configuration map `istio-cni-config-v2-3`.
```

Comment:
I don't know if "configuration map" is how you usually refer to ConfigMaps in the documentation, but I think replacing "configuration map" with ConfigMap would make this sentence much clearer to the average OpenShift user.

Reply:
Edit made